tests/validation/tests/v3_api/test_registry.py

```diff
@@ -201,17 +479,19 @@ def create_validate_workload(p_client, ns):
     workload = p_client.reload(workload)
 
     validate_workload(p_client, workload, "deployment", ns.name)
+    return workload
```

Comment:
Can we delete workload as part of this create_validate_workload method so that it doesn't have to be deleted for every test case ? Remove the deletes from each test case that uses this method.

Reply:
Done